.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"eslint.workingDirectories": [{ "mode": "auto" }]
```

Comment:
@edmundhung not sure if you use VSCode, but this tells the ESLint plugin to use each .eslintrc within its package.